addons/binding/org.openhab.binding.amazonechocontrol/README.md

```diff
@@ -0,0 +1,237 @@
+# Amazon Echo Control Binding
+
+This binding let control openHAB Amazon Echo devices (Alexa).
```

Comment:
I think the order of words is not correct here: 'This binding let control openHAB Amazon Echo devices'

Reply:
Sorry, my english is not best ;)

Reply:
No problem